druntime/test/shared/src/load.d

```diff
@@ -145,7 +138,7 @@ void main(string[] args)
     }
     else
     {
-        import core.sys.posix.dlfcn;
+        import core.sys.posix.dlfcn : dlopen, RTLD_LAZY, RTLD_NOLOAD;
         assert(dlopen(name.ptr, RTLD_LAZY | RTLD_NOLOAD) is null);
     }
```

Comment:
This is not in scope of this Pull Request, but this code is iffy. It assumes that 'not Windows' = posix and imports posix.dlfcn while it used to selectively import `RTLD_NOLOAD` from `core.sys.{dragonflybsd, linux, netbsd ...}.dlfcn`. It should probably do `else version (posix) { dlopen ... } else static assert(0, "unsupported platform")`.